tests/test_html_parser.py

```diff
@@ -14,14 +13,27 @@ class TestHTMLParser(TestCase):
     def test_basic_transformation(self):
         html = "<div><p>Hello</p></div>"
         result, _ = set_html_attributes(html, root_attributes=["data-root"], all_attributes=["data-all"])
-        expected = "<div data-root data-all><p data-all>Hello</p></div>"
-        assert result == expected
+
```

Comment:
I've updated the tests in `TestHTMLParser` in this file, to use Django's `assertHTMLEqual` instead of plain `assert`.